src/openforms/js/components/admin/form_design/registrations/zgw/ZGWOptionsFormFields.js

```diff
@@ -112,6 +127,10 @@ const ZGWFormFields = ({name, apiGroupChoices, confidentialityLevelChoices}) =>
             collapsible
             fieldNames={['objecttype', 'objecttypeVersion', 'contentJson']}
           >
+            <ObjectsAPIGroup
+              apiGroupChoices={objectsApiGroupChoices}
+              onApiGroupChange={onApiGroupChange}
+            />
```

Comment:
I think this one should be `isClearable` to disable objects API integration again? Though clearing the objecttype field has the same effect